proxy/src/main/java/io/openlineage/proxy/api/models/KafkaLineageStream.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.openlineage.proxy.api.models;
+
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import java.util.Properties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * KafkaLineageStream is responsible for sending open lineage events to kafka.  The collect() method is called each time an open lineage event is
+ * emitted by the data platform.
+ */
+public class KafkaLineageStream extends LineageStream
+{
+  private static final Logger log = LoggerFactory.getLogger(KafkaLineageStream.class);
+
+  private final String                   localServerId;
+  private final Properties               producerProperties;
+  private KafkaProducer<String, String>  producer;
+  private String                         topicName;
+
+  public KafkaLineageStream(String     localServerId,
```

Comment:
Mind if we define a `KafkaConfig` object? A class would make the configurable params more manageable as we add them.

```java
public KafkaLineageStream(KafkaConfig){...}
```